notebooks/examples/quickstart_in_5_minutes.ipynb

```diff
@@ -142,7 +142,16 @@
      "start_time": "2021-12-08T21:43:44.423627Z"
```

Comment:
<p>?</p><p>(fix error / disable warning for specific versions?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>